showcase/tslint.json

```diff
@@ -1,15 +0,0 @@
-{
-  "extends": [
```

Comment:
Why is this `tslint.json` file removed?

Reply:
Ooops! I thought I didn't take this change... My bad. I'll revert this